package.json

```diff
@@ -24,7 +24,7 @@
   "license": "MIT",
   "homepage": "https://github.com/companieshouse/overseas-entities-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.43",
+    "@companieshouse/api-sdk-node": "^2.0.41",
```

Comment:
did you mean to revert the api-sdk-node?

Reply:
Probably not. I'll change it back